src/components/Catchphrase/Catchphrase.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Catchphrase.css';
+
+export default function Catchphrase({
+  setCatchphrase,
+  catchphrase,
+  catchphrases,
+  setCatchphrases,
+}) {
+  const saveCatchphrase = () => {
+    setCatchphrases((prevState) => [...prevState, catchphrase]);
+  };
```

Comment:
On the saveCatchphrase function we can call setCatchphrase('') to reset the input when the button is clicked.